prebuild/action.yml

```diff
@@ -4,6 +4,9 @@ runs:
     - uses: actions/download-artifact@v3
     - run: $PACKAGE_MANAGER install --ignore-scripts
       shell: bash
+    - run: $PACKAGE_MANAGER install @napi-rs/cli@2.0.0
```

Comment:
I was having a problem with using the napi-rs/cli dependency in `execSync` within `index.js` ([here](https://github.com/DataDog/action-prebuildify/blob/b2f9272d4007c5e40d39ad4c4b4c51ec4cde98bc/prebuild/index.js#L119)) if the dependency was in package.json

Reply:
Was running into issues with `module not found` when installing it from the package.json, rather than directly here. Turns out it was an issue with the module not getting added to `PATH`, I've added a workaround to reference the path directly.

Reply:
putting @napi-rs/cli into devDependencies in `prebuild/package.json` and including it in the committed node_modules , referencing it with an absolute path into node_modules when we call it using execSync. 

I think including it in node_modules when it will only be used for napi-rs builds would be worse than manually installing it for napi-rs cases.

Reply:
Unless the dependency is really large I don't think it's a problem. Using the action will just pull a little bit more files, which is expected for JavaScript actions.

Reply:
took me a while due to an issue in windows with running the `@napi-rs/cli` command via an absolute path, but got it working in the end. The dependency is now is node_modules